Kickstarter-iOS/Views/Controllers/CheckoutViewController.swift

```diff
@@ -132,12 +132,6 @@ internal final class CheckoutViewController: DeprecatedWebViewController {
     self.present(vc, animated: true, completion: nil)
   }
 
-  fileprivate func goToSafariBrowser(url: URL) {
```

Comment:
There were 6 duplicated functions thorough our codebase...it should now be much DRYer